index.html

```diff
@@ -32,13 +34,124 @@ <h5>
+      <li><a href="#"><i class="fa-regular fa-hand-peace socialMedia"></i></a></li>
     </ul>
   </section>
+  <section class="work">
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio0.png" alt="" srcset="">
+      <h2>Tonic</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio1.png" alt="Project_Image" srcset="">
+      <h2>Multi-Post <br>
+      Stories</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio2.png" alt="Project_Image" srcset="">
+      <h2>Tonic</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio3.png" alt="Project_Image" srcset="">
+      <h2>Multi-Post<br>
+      Stories</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+  </section>
+  <section class="about_section">
+        <h2>About Myself</h2>
+        <p>
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] As part of the adjustment to the typography, kindly use the color `#344563` for this element. That way, it would be identical to the Figma template that you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219477646-a836772f-a714-4e06-a449-bf5fed0dafeb.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/219477730-e4c1e116-b82a-4510-aba5-fbb304e3e3cf.png)